src/lib/components/table/table-settings/table-settings.component.html

```diff
@@ -0,0 +1 @@
+<terra-button inputIcon="icon-settings" [tcTooltip]="'columnSettings.tooltip' | translate:_lang" (outputClicked)="_openSettings()"></terra-button>
```

Comment:
```suggestion
<terra-button inputIcon="icon-settings" [inputTooltipText]="'columnSettings.tooltip' | translate:_lang" (outputClicked)="_openSettings()"></terra-button>
```
Und ich glaube du musst das Testsetup jetzt noch etwas anpassen.

Reply:
Am besten mockst du den TerraButton.
```
@Component({
   selector: 'terra-button',
   template: ''
})
class MockButtonComponent extends TerraButtonComponent {}
```